### DISCUSSION TITLE
Have an undefined error.

### DISCUSSION THREAD

**Discussion by root-skaii** [vercel/community#1863] [Help] [answered]
### Summary

I've got undefined error, no lines provided.
If you know any method to fix it or how to find any error and their lines - I'll be happy.

On localhost it work very well, I used "Vercel CLI" and command to deploy host: "vercel dev".
Source from localhost and from site are the same, I uploaded it like 3-4 times and nothing changed.

### Additional information

500: INTERNAL_SERVER_ERROR Code: FUNCTION_INVOCATION_FAILED

I've checked logs but it just gave me:
python file (where happened error) and type of error
![изображение](https://user-images.githubusercontent.com/68514509/227417272-1038a19e-a8aa-461d-ac91-5af2d23e48b0.png)
![изображение](https://user-images.githubusercontent.com/68514509/227417347-5ca2ce96-614e-4117-a04d-af0f1481fdb7.png)
![изображение](https://user-images.githubusercontent.com/68514509/227417367-bdeff81a-0194-42ea-8f2b-a136e125e387.png)
![изображение](https://user-images.githubusercontent.com/68514509/227417390-eca0729f-8873-44cd-a975-03e6aabf52cf.png)

### Example

https://idv3h.vercel.app/

**Comment by root-skaii** [comment, marked as answer]
Nevermind, I fixed it when remaking Environment Settings and pressing every checkbox when saving.

**Reply by duongnguyen321** [reply]
can you show me a setting :(( 
or vercel.json.. 
pls :(
